ts/index.ts

```diff
@@ -674,10 +704,10 @@ class GossipSub extends BasicPubsub {
    * Returns the current time in milliseconds
    * @returns {number}
    */
-  _now () {
+  _now (): number {
     return Date.now()
   }
 }
 
-module.exports = GossipSub
-module.exports.multicodec = constants.GossipSubID
+module.exports = Gossipsub
```

Comment:
export = Gossipsub should be used in typescript
https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require